docker/prod/docker-compose.yml

```diff
@@ -22,6 +22,7 @@ services:
       dockerfile: ./packages/wallet/frontend/Dockerfile.prod
       args:
         PORT: ${WALLET_FRONTEND_PORT}
+        COOKIE_NAME: ${WALLET_BACKEND_COOKIE_NAME}
```

Comment:
should the name be prefixed with `WALLET_FRONTEND`?

Reply:
Used the same environment variable that we are using for the backend so we won't have two of them.